src/index.html

```diff
@@ -9,6 +12,23 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href=" "><img src="./images/logo.png" alt=" "></a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="item"><a class="link is-active" href=" ">Apple</a></li>
```

Comment:
Let's fix it everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">Apple</a>
          </li>
          
```